tests/test_fromFile.py

```diff
@@ -1,11 +1,10 @@
 from unittest import TestCase
-import os
 
 from mahjong.record.reader import from_file
 
 
 class TestFromFile(TestCase):
     def test_from_file(self):
-        filename = "tests\\2009060321gm-00b9-0000-75b25bcf.xml"
+        filename = "tests/2009060321gm-00b9-0000-75b25bcf.xml"
```

Comment:
for path， `os.path.join("test", "2009060321gm-00b9-0000-75b25bcf.xml")` may be a better solution.